cmd/secret/cmd/root.go

```diff
@@ -309,3 +395,35 @@ func (r *root) serverPingCmd() *cobra.Command {
 	serverPingCmd.Flags().DurationVarP(&timeout, "timeout", "t", 15*time.Second, "max request time to make a request. Default: '15 seconds'")
 	return serverPingCmd
 }
+
+func migrateUp(postgres, source string, logger *zap.SugaredLogger) error {
+	logger.Info("root: starting migrate up")
+	m, err := migrate.New(
+		source,
+		postgres)
+	if err != nil {
+		return err
+	}
+	logger.Info("root: migrate created")
+	err = m.Up()
+	if err != nil {
+		return err
+	}
+	logger.Info("root: migrate up")
+	return nil
```

Comment:
It would be more stylish to use `Named()` for such things.
```suggestion
        logger = logger.Named("migration")
	logger.Info("starting from source=%s", source)
	m, err := migrate.New(
		source,
		postgres)
	if err != nil {
		return err
	}
	logger.Info("created")
	err = m.Up()
	if err != nil {
		return err
	}
	logger.Info("finished")
	return nil
```